packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -106,6 +110,7 @@ type PortfolioKitState = {
   accounts: MapStore<SupportedChain, AccountInfo>;
   positions: MapStore<PoolKey, Position>;
   nextFlowId: number;
+  targetAllocation: TargetAllocation;
```

Comment:
I'm afraid this is a breaking change.

IOU an upgrade test to confirm. cc @LuqiPan